### PR TITLE
Modify Note in MRT Core overview page.

### PR DESCRIPTION
**Pull request by rohanp-msft**
Update Note with latest.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@rohanp-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/windows-app-sdk/mrtcore/mrtcore-overview.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ A PRI file contains actual string resources. Embedded binary and file path resou
 PRI files contain only data, so they don't use the portable executable (PE) format. They are specifically designed to be data-only.
 
 > [!NOTE]
-> In Windows App SDK version 0.8, the **Build Action** for resources is automatically set, reducing the need for manual project configuration.
+> For .NET apps, in Windows App SDK version 0.8 and onward, the **Build Action** file property for resource files in Visual Studio is automatically set, reducing the need for manual project configuration. Version 1.0 introduced [issue 1786](https://github.com/microsoft/WindowsAppSDK/issues/1786). This is fixed in 1.1 Stable, but the fix requires .NET SDK 6.0.300 or higher. If you're using a lower version of the .NET SDK, please continue to use the workaround in the 1.0 Stable release notes.
```
Comment (huichen123) [inline hub/apps/windows-app-sdk/mrtcore/mrtcore-overview.md]: >1786 [](http://example.com/codeflow?start=235&length=4)

nit: as this is resolved as dupe to 1674, consider using the primary issue.

**Comment by AnnaMHuff** [comment]
@hickeys - Can you review the proposed changes?

*IMPORTANT*: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge.

Thanks!

#label:"aq-pr-triaged"

**Comment by PRMerger17** [comment]
@rohanp-msft : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/windows-app-sdk/mrtcore/mrtcore-overview.md (hunk 1 above)
Comment (rohanp-msft) [reply hub/apps/windows-app-sdk/mrtcore/mrtcore-overview.md]: Done

**Review by hickeys — APPROVED** [review]

**Pull request merged**